SaintCoinach/Definitions/Stain.json

```diff
@@ -17,7 +17,7 @@
       "name": "SubOrder"
     },
     {
-      "index": 3,
+      "index": 4,
```

Comment:
Both 3 and 4 are using the same value, it's both name, the only difference seems to be with second one being maybe for a different usage, as it's sometimes different in other languages.

Anyways, 3 is still Name regardless.

Reply:
ffxiv_dx11.exe ver. 2023.07.26.0000.0000: 141201F90
col 3: offset 0
col 4: offset 4
![image](https://github.com/xivapi/SaintCoinach/assets/64143453/5ca9e336-e36e-4ed3-a1a0-d8574613514a)

ffxiv_dx11.exe ver. 2023.03.24.0000.0000: 14121D200
col 3: offset 0
![image](https://github.com/xivapi/SaintCoinach/assets/64143453/592c5133-f0cf-4095-896c-93396b9cf5bd)

Reply:
and i dont find any reference in new code keep using offset 0